app/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,23 @@
   <string name="app_name">Oppia</string>
   <string name="welcome_text">Welcome to Oppia!</string>
   <string name="welcome_back_text">Welcome back to Oppia!</string>
+  <string name="title_activity_navigation_drawer">NavigationDrawerActivity</string>
```

Comment:
A lot of these strings aren't being used. Please remove all unused strings.

Reply:
Removed